Packs/Fortimail/Integrations/Fortimail/Fortimail.yml

```diff
@@ -0,0 +1,2214 @@
+    arguments:
+    - name: name
+      description: The name of the anti spam GeoIP group.
+      default: true
+    - name: all_results
+      description: Whether to retrieve all results by overriding the default limit.
+      defaultValue: 'false'
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
+    - name: limit
+      description: The maximum number of records to retrieve.
+      defaultValue: '50'
+    outputs:
+    - contextPath: FortiMail.GeoIPgroup.mkey
+      description: The name of the GeoIP group.
+      type: String
+    - contextPath: FortiMail.GeoIPgroup.country
+      description: A list of the GeoIP group countries.
+      type: Number
+    - contextPath: FortiMail.GeoIPgroup.comment
+      description: A brief comment for the GeoIP group countries.
+      type: String
+    - contextPath: FortiMail.GeoIPgroup.is_referenced
+      description: Number of referencing entities.
+      type: Number
+  - name: fortimail-antispam-profile-list
+    description: List AntiSpam profiles. Antispam profiles are sets of antispam scans that you can apply by selecting one in a policy. if an name is given, the command will return the information about the wanted AntiSpam profile. Mainly used in the configuration of IP policy.
+    arguments:
+    - name: name
+      description: The name of the anti spam profile.
+      default: true
+    - name: all_results
+      description: Whether to retrieve all results by overriding the default limit.
+      defaultValue: 'false'
+      auto: PREDEFINED
+      predefined:
+      - 'true'
+      - 'false'
+    - name: limit
+      description: The maximum number of records to retrieve.
+      defaultValue: '50'
+    outputs:
+    - contextPath: FortiMail.AntispamProfile.mkey
+      description: The name of the anti spam profile.
+      type: String
+    - contextPath: FortiMail.AntispamProfile.dictionary_type
+      description: The type of the dictionary type.
+      type: Number
+    - contextPath: FortiMail.AntispamProfile.minimum_dictionary_score
+      description: The minimu, number of the dictionary score.
+      type: Number
+    - contextPath: FortiMail.AntispamProfile.comment
+      description: A brief comment for the anti spam profile.
+      type: String
+    - contextPath: FortiMail.AntispamProfile.isReferenced
+      description: Number of referencing entities.
+      type: Number
+  - name: fortimail-antivirus-profile-list
+    description: List AntiVirus profiles. if the FortiMail unit detects a virus, it will take actions as you define in the antivirus action profiles. if an name is given, the command will return the information about the wanted AntiVirus profile. Mainly used in the configuration of IP policy.
```

Comment:
```suggestion
    description: List AntiVirus profiles. If the FortiMail unit detects a virus, it will take actions defined in the antivirus action profiles. If a name is given, the command will return the information about the specified AntiVirus profile. Mainly used in the configuration of IP policy.
```